### ISSUE TITLE
Support for 'viewScript' in block.json

### ISSUE THREAD

**Issue by KBraNet** [evo-mark/vite-plugin-gutenberg-blocks#2]
https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#view-script
When I add "viewScript" to block.json I get this error:
`Notice: Function register_block_script_handle was called incorrectly. The asset file (/var/www/html/wp-content/plugins/gutenberg-blocks/build/accordion/frontend/index..asset.php) for the "viewScript" defined in "frontkom/accordion" block definition is missing. Please see [Debugging in WordPress](https://wordpress.org/documentation/article/debugging-in-wordpress/) for more information. (This message was added in version 5.5.0.) in /var/www/html/wp-includes/functions.php on line 5865`
When I manually add `view.js` and `view.asset.php` in the build folder, everything works fine

```
\\ view.asset.php
<?php return ["dependencies" => [], "version" => "d38e695eab960db2e97e7f9824b5f420"];
```
So I would need to add to the configuration the ability to add an additional entry point for front end scripts and generate a view.asset.php file for it

**Comment by krzysiek0912** [comment]
@craigrileyuk I almost found a solution. The problem, however, is the format. If we have set anything other than “iife”, the code doesn’t work in the browser. However, this format does not support code-splitting builds
 https://github.com/evo-mark/vite-plugin-gutenberg-blocks/compare/main...KBraNet:vite-plugin-gutenberg-blocks:feature/add-frontend-scripts

**Comment by craigrileyuk** [comment]
Closed with #a158316 (released via NPM as v1.3.1)
